subprojects/persistent-cache/src/integTest/groovy/org/gradle/cache/internal/DefaultFileLockManagerContentionIntegrationTest.groovy

```diff
@@ -258,7 +259,7 @@ class DefaultFileLockManagerContentionIntegrationTest extends AbstractIntegratio
 
             class ToolSetupRunnable implements Runnable {
                 void run() {
-                    CacheRepository cacheRepository = ZincCompilerServices.getInstance(new File("home")).get(CacheRepository.class);
+                    CacheRepository cacheRepository = ZincCompilerServices.getInstance(new File(URI.create("${gradleUserHome.absoluteFile.toURI()}"))).get(CacheRepository.class);
```

Comment:
We have a method in `TextUtil` that escapes a String for use in Java code.

Reply:
Thanks, done in f7aadc0d4c5.